src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -315,6 +317,9 @@ public function load(array $configs, ContainerBuilder $container)
         $container->registerForAutoconfiguration(ObjectInitializerInterface::class)
             ->addTag('validator.initializer');
 
+        $container->registerForAutoconfiguration(LoggerAwareInterface::class)
+            ->addMethodCall('setLogger', array(new Reference(LoggerInterface::class)));
```

Comment:
> Calling a setter twice is not an issue to me so not really a pb.

It would be a WTF no? Requires to move your definition outside the current file/context at least... not sure that justifies our technical implem, which is correct; we _add_ a method call. But being a setter that's messes things up.

Also not sure it justifies a `addMethodCallOnce` feat., perhaps as 2nd arg?